octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -53,6 +58,15 @@ def set_context_object(ctx: click.Context, airbyte_url: str, workspace_id: str,
     return ctx
 
 
+def assign_api_values(api_client: airbyte_api_client.ApiClient, api_headers: Optional[List[ApplicationHeader]]) -> NoReturn:
+    if api_headers:
```

Comment:
I'd make this check upstream and call this function only if api_headers are set.

Reply:
@Imbruced please let me know what you think about this suggestion.